### DISCUSSION TITLE
Issue for two radionuclides of the JEFF3.3 database

### DISCUSSION THREAD

**Discussion by Frozaccio** [njoy/NJOY2016#341] [Q&A]
Dear Njoyer,

I am using NJOY2016 to generate a cross section database from JEFF3.3. Everything goes well except for two nuclides which are 6-C-13g.jeff33 and 8-O-17g.jeff33.

Below you will find the input I use for C-13 as well as the error message:

**NJOY INPUT**
`reconr / %%%%%%%%%%%%%%%%%%% Reconstruct XS for neutrons %%%%%%%%%%%%%%%%%%%%%%%
20 21
'JEFF-3.3 PENDF for   6-C - 13 '/
628 2/
0.001/ err
'JEFF-3.3:   6-C - 13 '/
'Processed by NJOY'/
0/

broadr / %%%%%%%%%%%%%%%%%%%%%%% Doppler broaden XS %%%%%%%%%%%%%%%%%%%%%%%%%%%%
20 21 22
628 1 0 0 0. /
0.001/ errthn
293
0/

heatr / %%%%%%%%%%%%%%%%% Add heating kerma (local photons) %%%%%%%%%%%%%%%%%%%%
20 22 23 /
628 4 0 0 1 /
302 318 402 444 /

heatr / %%%%%%%%%%%%%%%%%%%%%%%%% Add heating kerma %%%%%%%%%%%%%%%%%%%%%%%%%%%%
20 22 24 /
628 4 0 0 0 /
302 318 402 444 /

gaspr / %%%%%%%%%%%%%%%%%%%%%%%%% Add gas production %%%%%%%%%%%%%%%%%%%%%%%%%%%
20 24 25 /

purr / %%%%%%%%%%%%%%%%%%%%%%%% Add probability tables %%%%%%%%%%%%%%%%%%%%%%%%%
20 25 26 /
628 1 1 20 64 /
293
1.e10
0/

acer / %%%%%%%%%%%%%%%%%%%%%%%% Write out in ACE format %%%%%%%%%%%%%%%%%%%%%%%%
20 26 0 27 28
1 0 1 .01 /
'JEFF-3.3:   6-C - 13  at 293'/
628 293
1 1 1/
/
stop`

**Error message for both nuclides**

`njoy 2016.69  07Feb22                                       07/11/24 13:33:37
 *****************************************************************************

 reconr...                                                                0.0s

 ---message from rdf2bw---calculation of angular distribution not installed.

 broadr...                                                                0.0s

 heatr...                                                                 0.1s

 ---message from hinit---mf6, mt102 does not give recoil za=  6014
                          photon momentum recoil used.

 ***error in sixbar***awp is zero for for mt 29 particle  2005

STOP 77
ERROR: Command '/usr/local/NJOY2016/install/bin/njoy' returned non-zero exit status 77.

 njoy 2016.69  07Feb22                                       07/11/24 13:33:38
 *****************************************************************************

 reconr...                                                                0.0s

 ---message from rdf2bw---calculation of angular distribution not installed.

 broadr...                                                                0.1s

 heatr...                                                                 0.1s

 ---message from hinit---mf6, mt102 does not give recoil za=  8018
                          photon momentum recoil used.

 ***error in sixbar***awp is zero for for mt109 particle  2006

STOP 77
ERROR: Command '/usr/local/NJOY2016/install/bin/njoy' returned non-zero exit status 77.
`

Do you know what goes wrong? I do not understand the meaning of the error message ` ***error in sixbar***awp is zero for for mt109 particle  2006` and have no idea how to solve it.

I would appreciate some help, please.
Thanks in advance.
Thomas

**Comment by whaeck** [comment]
AWP is an ENDF variable stored on the multiplicity record for each reaction product in MF6. It represents the atomic mass given in neutron mass units and is used during processing. For any particle other than photons, it cannot be zero. It would appear that for the evaluations you are using, those values are set to zero - which causes the error messages.

The solution is quite simple: check the AWP value for He5 and He6 in a few other evaluations (or use some other source) and replace the zero values in the evaluation. Note that AWP is the atomic mass in NEUTRON mass units (and not atomic mass units as one would expect).

**Comment by Frozaccio** [comment]
Thank you very much @whaeck. That was exactly the issue. I find this strange since I have downloaded the JEFF3.3 evaluation from [oecd-nea_jeff3.3](https://www.oecd-nea.org/dbdata/JEFF33/ ). That means the AWP of He-5 and He-6 is missing inside those files and they probably noticed the missing value when they processed the library.
